test/ch/aggregation_test.exs

```diff
@@ -51,10 +51,14 @@ defmodule Ch.AggregationTest do
       insert into candle_fragments
         (ticker, time, high, open, close, low)
       VALUES
-      ('INTC', '2023-04-13 20:33:00', 32, 32, 32, 32),
-      ('INTC', '2023-04-13 20:34:00', 33, 33, 33, 33),
-      ('INTC', '2023-04-13 20:35:00', 32, 32, 31, 26),
-      ('INTC', '2023-04-13 20:36:00', 32, 27, 27, 27)
+      -- 1681410780 '2023-04-13 20:33:00'
+      -- 1681410840 '2023-04-13 20:34:00'
+      -- 1681410900 '2023-04-13 20:35:00'
+      -- 1681410960 '2023-04-13 20:36:00'
+      ('INTC', 1681410780, 32, 32, 32, 32),
+      ('INTC', 1681410840, 33, 33, 33, 33),
+      ('INTC', 1681410900, 32, 32, 31, 26),
+      ('INTC', 1681410960, 32, 27, 27, 27)
```

Comment:
Maybe we can assert on `20:00` and `21:00` https://github.com/Zarathustra2/ch/pull/1#discussion_r1167965992?

Why does it have to be `18:00` and `19:00` in the results?

Reply:
argg why are DateTimes so hard. Maybe unix timestamp is better with a comment? 

Try my latest push. Fails locally for me but might be fine for CI but still the same issue I think, not sure what is going on tbh